source/controller/wizard_controller.py

```diff
@@ -166,6 +166,14 @@ def verify_and_save_project_folder(self):
             self.update_complete_project_files_page(complete_project_location=False)
 
     def save_project_description(self):
+        """Takes a 'str' added by the user in the wizard 'project_description_input',
+         save in the 'wizard_information_store' and set the 'project_description' in the review page.
```

Comment:
This method does not take a string, as none is passed in the parameter.  It is not a good practice to mention the internal variables the method uses, so no 'wizard_information_store', for example. 

```suggestion
        """Updates the project description throughout the places it is being used
```
Maybe it is even a good idea to change the method name to `update_project_description`, given what it does.